.github/actions/start-opensearch-with-one-plugin/action.yml

```diff
@@ -89,14 +89,14 @@ runs:
 
     - name: Run OpenSearch with plugin on Windows
       if: ${{ runner.os == 'Windows'}}
-      run: start .\opensearch-${{ inputs.opensearch-version }}-SNAPSHOT\bin\opensearch.bat
+      run: start-process .\opensearch-${{ inputs.opensearch-version }}-SNAPSHOT\bin\opensearch.bat
       shell: pwsh
 
     # Give the OpenSearch process some time to boot up before sending any requires, might need to increase the default time!
     - name: Sleep while OpenSearch starts
       uses: peternied/action-sleep@v1
       with:
-        seconds: 30
+        seconds: 60
```

Comment:
Pretty sure we can go back to 30

#### Win JDK11
14 seconds to start
```
[2022-12-07T17:53:47,980][INFO ][o.o.n.Node               ] [fv-az89-842] version[3.0.0-SNAPSHOT], pid[1064], build[zip/5500114cc55d94bb8a7699f3db6870d83dc92445/2022-12-07T01:33:59.605829500Z], OS[Windows Server 2022/10.0/amd64], JVM[Azul Systems, Inc./OpenJDK 64-Bit Server VM/11.0.17/11.0.17+8-LTS]
...
[2022-12-07T17:54:01,725][INFO ][o.o.s.c.ConfigurationRepository] [fv-az89-842] Node 'fv-az89-842' initialized
```

#### Win JDK17
12 seconds to start
```
[2022-12-07T17:53:21,015][INFO ][o.o.n.Node               ] [fv-az282-542] version[3.0.0-SNAPSHOT], pid[6536], build[zip/5500114cc55d94bb8a7699f3db6870d83dc92445/2022-12-07T01:33:59.605829500Z], OS[Windows Server 2022/10.0/amd64], JVM[Azul Systems, Inc./OpenJDK 64-Bit Server VM/17.0.5/17.0.5+8-LTS]
...
[2022-12-07T17:53:33,835][INFO ][o.o.s.c.ConfigurationRepository] [fv-az282-542] Node 'fv-az282-542' initialized
```